jre8/plan.ps1

```diff
@@ -0,0 +1,34 @@
+$pkg_name="jre8"
+$pkg_origin="core"
+$pkg_version="8.172.0"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=@("Apache-2.0")
+$pkg_upstream_version="8u172"
+$pkg_source="http://download.oracle.com/otn-pub/java/jdk/$pkg_upstream_version-b11/a58eab1ec242421181065cdc37240b08/jre-$pkg_upstream_version-windows-i586.exe"
+$pkg_filename="$pkg_name-$pkg_version.exe"
+$pkg_shasum="434d7cbe88e2a28ee37c7345b5d93810c98112908f7dd5a273a0611482898928"
+$pkg_bin_dirs=@("bin")
+
+function Invoke-Download() {
+     $Cookie  = New-Object -TypeName System.Net.Cookie
+     $Cookie.Domain = 'oracle.com'
+     $Cookie.Name   = 'oraclelicense'
+     $Cookie.Value  = 'accept-securebackup-cookie'
+     $Session = New-Object -TypeName Microsoft.PowerShell.Commands.WebRequestSession
+     $Session.Cookies.Add($Cookie)
+     try {
+        $bar = Invoke-WebRequest -Uri $pkg_source -WebSession $Session -OutFile $HAB_CACHE_SRC_PATH/$pkg_filename
```

Comment:
What is `$bar` assigned for?

Reply:
Whoop, was using it to get the redirect. But ended up with the try catch. I’ll get this cleaned up!